classify/trainingsampleset.cpp

```diff
@@ -507,22 +507,6 @@ bool TrainingSampleSet::DeleteableSample(const TrainingSample* sample) {
   return sample == NULL || sample->class_id() < 0;
 }
 
-static Pix* DebugSample(const UNICHARSET& unicharset,
```

Comment:
The all 'classify' directory will most likely be removed by Ray (it is part of the old engine). There is no point in applying any change to files under this directory.